engine/common/ios/AsyncSocket.h

```diff
@@ -0,0 +1,282 @@
+//
```

Comment:
Platform-specific stuff shouldn't go in `engine/common/`. 

Move it under `platform/ios/`.